CHANGELOG.md

```diff
@@ -20,6 +20,14 @@ per `completion-cycle-threshold`) now ([#419], [#456]).
 [#455]: https://github.com/raxod502/selectrum/pull/455
 [#456]: https://github.com/raxod502/selectrum/pull/456
 
+### Bugs fixed
+When doing `completing-read-multiple`, selecting an additional
+candidate and exiting by pressing <kbd>RET</kbd> no longer fails with
+`(wrong-type-argument char-or-string-p nil)` when there are existing
+candidates already selected using <kbd>TAB</kbd> ([#460]).
```

Comment:
Thanks! Some minor nitpicks: We used `RET` and `TAB` elsewhere in this document, so better to keep it consistent and as the description is for users it isn't necessary to document the exact error describing that it failed before and is fixed now is enough.

Reply:
No worries. How does it look now?